dask/array/reductions.py

```diff
@@ -264,7 +266,9 @@ def partial_reduce(func, x, split_every, keepdims=False, dtype=None, name=None,
     if np.isscalar(meta):
         return Array(graph, name, out_chunks, dtype=dtype)
     else:
-        return Array(graph, name, out_chunks, meta=meta.astype(dtype))
+        with ignoring(AttributeError):
```

Comment:
Now a noob python question: is that equivalent to `try ... except AttributeError: pass`?

Reply:
Yes, this actually isn't a Python thing, it's a Dask thing

https://github.com/dask/dask/blob/e32fbe35704976600233191ea03dd73a3dec5a88/dask/utils.py#L78-L83

(though I think we implemented it only after seeing that something similar was planned for Python)

Reply:
Ah, that's a Dask util function, and I assumed correctly. So ignore this. :)

Reply:
Sorry, didn't see your comment before. What is planned for Python, is there a PEP for that or something already integrated?

Reply:
I don't know.  I didn't follow the progress here after I saw some comment about it.